perf/specjbb/build.xml

```diff
@@ -0,0 +1,66 @@
+<?xml version="1.0"?>
+
+<!--
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+-->
+
+<project name="Specjbb suite" default="build" basedir=".">
+	<taskdef resource="net/sf/antcontrib/antlib.xml" />
+	<description>Specjbb suite</description>
+
+	<!-- set global properties for this build -->
+	<property name="DEST" value="${BUILD_ROOT}/perf/specjbb" />
+	<property name="SRC" location="." />
+	
+	<target name="init">
+		<mkdir dir="${DEST}" />
+	</target>
+
+	<target name="getSpecjbbSuite" depends="init">
+        
+        <property environment="env"/>
+        <property name="SPECJBB_DEST" value="${DEST}/suite" />
+        <property name="SPECJBB_SRC" value="${env.SPECJBB_SRC}" />
+        <property name="JAVA_BIN" value="${env.TEST_JDK_HOME}/bin/java"/>
+        <property name="SPECJBB_JAR" value="specjbb2015.jar"/>
+        <property name="SCRIPT_LOG" value="${DEST}/getSpecjbbSuite.log"/>
+        <property name="SCRIPT_SUCCESS" value="0"/>
+
+        <exec executable="bash" failifexecutionfails="true" output="${SCRIPT_LOG}" resultproperty="SCRIPT_RESULT">
+            <env key="SPECJBB_DEST" value="${SPECJBB_DEST}"/>
+            <env key="SPECJBB_SRC" value="${SPECJBB_SRC}"/>
+            <env key="JAVA_BIN" value="${JAVA_BIN}"/>
+            <env key="SPECJBB_JAR" value="${SPECJBB_JAR}"/>
+            <arg value="${SRC}/build/getSpecjbbSuite.sh"/>
+		</exec>
+        
+        <fail message="Failed to get SPECjbb suite. See the following file for more info: ${SCRIPT_LOG}">
+            <condition>
+                <not>
+                    <equals arg1="${SCRIPT_RESULT}" arg2="${SCRIPT_SUCCESS}"/>
+                </not>
+            </condition>
+        </fail> 
+        
+	</target>	
+
+	<target name="dist" depends="getSpecjbbSuite" description="generate the distribution">
+        <copy todir="${DEST}/run">
```

Comment:
nit: consider using only 1 type of spacing (tabs?) in these files for consistency on disk and in visual formatting.